### PR TITLE
implementation of stack using Queue in Java

### PR DESCRIPTION
**Pull request by ritvij14**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description
This is the code for the implementation of a stack using 2 queues.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Issue #2244 
(Write your answer here.)

### CODE REVIEW

**Comment by ritvij14** [comment]
@siddharth25pandey @HarshCasper @iamrajiv  please review this.

File: Java/ds/StackUsingQueue.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+ * 5
+ * 4
+ * 3
+ * 2
+ * 1
+ * Current Size:0
+ * 
+ * Time complexity: O(n) for push, O(1) for pop
+ * Space complexity: O(n)
+ */
```
Comment (HarshCasper) [inline Java/ds/StackUsingQueue.java]: Add a new line at the EOF

File: Java/ds/StackUsingQueue.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+        int size() {
+            return currentSize;
+        }
+    }
+
+    public static void main(String[] args) {
+        Stack stack = new Stack();
+
+        // taking input
```
Comment (HarshCasper) [inline Java/ds/StackUsingQueue.java]: Redundant Comment

File: Java/ds/StackUsingQueue.java [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter size:");
+        int size = sc.nextInt();
+        System.out.println("Enter the numbers:");
+        for (int i = 0; i < size; i++) {
+            stack.push(sc.nextInt());
+        }
+        sc.close();
+
+        // giving output
```
Comment (HarshCasper) [inline Java/ds/StackUsingQueue.java]: Redundant Comment

File: Java/ds/StackUsingQueue.java [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            currentSize = 0;
+        }
+
+        void push(int num) {
+            currentSize++;
+
+            // first, push num into an empty qu2
+            qu2.add(num);
+
+            // now we push the other elements from qu1 to qu2
```
Comment (HarshCasper) [inline Java/ds/StackUsingQueue.java:25]: Add similar comments to other functionality

Comment (ritvij14) [reply Java/ds/StackUsingQueue.java:25]: Done

File: Java/ds/StackUsingQueue.java (hunk 3 above)
Comment (ritvij14) [reply Java/ds/StackUsingQueue.java]: Removed

File: Java/ds/StackUsingQueue.java (hunk 2 above)
Comment (ritvij14) [reply Java/ds/StackUsingQueue.java]: Removed

File: Java/ds/StackUsingQueue.java (hunk 1 above)
Comment (ritvij14) [reply Java/ds/StackUsingQueue.java]: Done

**Comment by ritvij14** [comment]
@HarshCasper changes done :+1:

**Review by HarshCasper — APPROVED** [review]

**Review by siddharth25pandey — APPROVED** [review]
@ritvij14 Good to go. Able to merge 👍

**Review by iamrajiv — APPROVED** [review]

**Comment by ritvij14** [comment]
@iamrajiv please add a label from this so that it gets counted in the leaderboard. Right now points for this have not been added.
image

**Pull request merged**